src/crimson/common/errorator.h

```diff
@@ -859,6 +860,45 @@ struct errorator {
     return make_ready_future<>();
   }
 
+  template <typename Iterator, typename Func>
+  static inline errorator<AllowedErrors...>::future<>
+  parallel_for_each(Iterator begin, Iterator end, Func&& func) noexcept {
+    parallel_for_each_state* s = nullptr;
+    // Process all elements, giving each future the following treatment:
+    //   - available, not failed: do nothing
+    //   - available, failed: collect exception in ex
+    //   - not available: collect in s (allocating it if needed)
+    while (begin != end) {
+      auto f = seastar::futurize_invoke(std::forward<Func>(func), *begin);
+      begin++;
+      if (!f.available() || f.failed()) {
+	if (!s) {
+	  using itraits = std::iterator_traits<Iterator>;
+	  auto n = (seastar::internal::iterator_range_estimate_vector_capacity(
```

Comment:
@xxhdx1985126 could you use spaces instead of tabs for indent in this file? to be more consistent with the rest part of this source file. also change `indent-tabs-mode:t` to `indent-tabs-mode:nil`, and change `smarttab` to `smarttab expandtab`  in the header?

Reply:
ok